Converted WP/PHP files to Static HTML

albrechtnate opened the pull request:
Resolves #2

albrechtnate committed:
First pass at converting PHP files to Static HTML

albrechtnate committed:
Completed WP/PHP to Static HTML Coversion

albrechtnate committed:
Removed the Docs folder which had the WP/SQL content since it is now integrated into the HTML.

albrechtnate committed:
Renamed CSS File and Updated HTML Links to Match

albrechtnate committed:
First pass at Sass Conversion

• Created styles.scss
• Broke styles.scss up into a number of partials
• Created variables for the colors and some breakpoints

philschanely committed:
Reorganize into src and add npm pagackage with scripts

philschanely commented:
Good job organizing styles. However, I'd like to see you make use of the Sass features like nesting and mixins to reduce repetition in your styles. 

Also consider whether you can use `span` instead of `li` for your bullet separators or make use of `::after` to avoid this markup altogether. 

I've moved all your site files into a `src` folder in the latest commit and added `package.json`. Now you can run `npm install` one time to install all the dependencies and then any time afterwards use `npm start` to batch run browser-sync and sass compilation. 

Comment when updates are made and I'll review again. 

Thanks!

albrechtnate committed:
Package Lock update as a result of running npm install

albrechtnate committed:
Nested selectors in all the files and added one mixin

albrechtnate committed:
More Vars, Px->Rem Function, Typography Mixin

- Created vars so I define the path to the image folder and font folder once
- Created a function to convert pixels to rems
- Created a mixin that would take a pixel font size and line-height and create rem and unitless versions
- Converted my existing rem values back into pixels and implemented the typography mixin

albrechtnate committed:
Converted li.separator in menus to ::after as noted in Pull Request #1 Comment

albrechtnate commented:
@philschanely I’ve updated the Sass files to take advantage of the nesting and mixin features. I also converted the li.separator markup in the menus into a css pseudo class. I think I am ready for this pull request to be merged.

philschanely reviewed (approved):
Fantastic work with the suggestions and more. You've got the idea for sure!
